src/sqlfluff/cli/commands.py

```diff
@@ -421,7 +421,7 @@ def fix(force, paths, parallel, bench=False, fixed_suffix="", logger=None, **kwa
     formatter.dispatch_config(lnt)
 
     # Set up logging.
-    set_logging_level(verbosity=verbose, logger=logger)
+    set_logging_level(verbosity=verbose, logger=logger, stderr_output=fixing_stdin)
```

Comment:
For consistency, should we consider making `lint` and `parse` behave the same as `fix`?

Reply:
They're already set up for this! They both send logs to stderr if the output is meant to be machine readable (json, yaml), so that's already covered.